test/gtest/ucp/test_ucp_stream.cc

```diff
@@ -341,8 +341,10 @@ void test_ucp_stream::do_send_exp_recv_test(ucp_datatype_t datatype)
 {
     const size_t dt_elem_size = UCP_DT_IS_CONTIG(datatype) ?
                                 ucp_contig_dt_elem_size(datatype) : 1;
-    const size_t msg_size = dt_elem_size * UCS_MBYTE;
-    const size_t n_msgs   = 10;
+    const size_t msg_size     = dt_elem_size *
+                                static_cast<int>(UCS_MBYTE /
+                                                 ucs::test_time_multiplier());
```

Comment:
maybe just `int(UCS_MBYTE / ucs::test_time_multiplier())`?

Reply:
why `static_cast<int>()` is not ok here?

Reply:
it's OK but `int()` looks simpler and does the same

Reply:
done